hrjobcontract/js/test/controllers/modal/modal-contract-new.controller.spec.js

```diff
@@ -216,6 +223,31 @@ define([
       });
     });
 
+    describe('when user clicks on the "Provider options" wrench icon', function () {
+      providersPopupLists = [
```

Comment:
this var is only used in this particular `describe`, so then you shouldn't declare it at the top of the test file, declare it here
```js
describe('when user clicks on the "Provider options" wrench icon', function () {
  var providersPopupLists;
```
also, don't assign values outside of `beforeEach`, otherwise you can't be sure that the state will be fully reset on every test run

```js
describe('....', function () {
  var providersPopupLists;

  beforeEach(function () {
    providersPopupLists = { /* .. */ };
  });
});
```

Reply:
Noted. This will be updated.